api/types.go

```diff
@@ -795,12 +790,10 @@ func (pin *Pin) ProtoUnmarshal(data []byte) error {
 	// pin.UserAllocations = opts.GetUserAllocations()
 	t := opts.GetExpireAt()
```

Comment:
Does this ever return nil?

Reply:
if time is zero(uninitialized), we are keeping protobuf timestamp as nil, so yes.

I can pass uninitialized time, but this saves some memory.